doc/lightning-batching.7.md

```diff
@@ -0,0 +1,54 @@
+lightning-batching -- Command to allow database batching.
```

Comment:
How is this safe? There are a number of places where we assume that the DB transaction has been unrevocably been committed to the DB, i.e., HTLC changes. If we permit a DB transaction to linger for longer than the synchronous JSON RPC function, then we cannot guarantee that anymore, or if so how? My fear is that if we start a batch at an inopportune time, then commit an update to a channel, leaving the DB tx dangling, and then kill the daemon, we will have just forgotten about a commit that we should absolutely not forget.

Reply:
Notice that this would not be such an issue if we had support for parallel transactions, where we could have a transaction for RPC and have that distinct from the state machine transaction, but alas sqlite3 does not allow us to do that.

Reply:
No, we always close the transaction before blocking. We only batch when we read more than one command at once (which happens *a lot* with autoclean slamming us with deletes).

Enabling batching does not *guarantee* batching: that would be hard to do, indeed!